README.md

```diff
@@ -66,7 +66,7 @@ $ puppet-query 'nodes[certname]{}'
     "certname" : "foo.example.com"
   }
 ]
-$ puppet-db status
+$ puppet-db status # Note `puppet-db status` will be released in puppet-client-tools 1.1.0
```

Comment:
Is this needed? Seems like it's assumed that if someone is looking at the readme in master, it could have stuff in it that hasn't been released yet.